ibm/resource_ibm_iam_access_group_test.go

```diff
@@ -94,7 +94,7 @@ func testAccCheckIBMIAMAccessGroupDestroy(s *terraform.State) error {
 		// Try to find the key
 		_, _, err := accClient.AccessGroup().Get(agID)
 
-		if err != nil && !strings.Contains(err.Error(), "404") {
+		if err == nil || !strings.Contains(err.Error(), "404") {
 			return fmt.Errorf("Error waiting for access group (%s) to be destroyed: %s", rs.Primary.ID, err)
```

Comment:
We need to guard against referencing `err` here, since it might be `nil`.  I'll fix that.